examples/learn/agents/slack_agent.py

```diff
@@ -34,18 +36,20 @@ class MiraBot(BaseModel):
         {input}
         """
     )
-    async def _step(self, input: str): ...
+    async def _step(self, input: str) -> None: ...
 
-    async def register_event_handlers(self):
+    async def register_event_handlers(self) -> None:
         @self.app.event("message")
-        async def handle_message_events(message, say, logger):
+        async def handle_message_events(
+            message: dict, say: Callable, logger: Any
```

Comment:
does the decorator not indicate what these types should actually be rather than just `Callable` and `Any`?